yuptoo/processor/report_processor.py

```diff
@@ -21,65 +21,58 @@
 FAILURE_CONFIRM_STATUS = 'failure'
 
 
-def process_report(consumed_message, p, request_obj):
-    global producer
-    producer = p
-    report_tar = download_report(consumed_message)
-    report_json_files = extract_report_slices(report_tar, request_obj)
-
+def process_report_slice(report_slice, request_obj):
```

Comment:
@patilsuraj767  - As you are changing the existing report processing, could you please add description of what the method is and what it's used for?

Reply:
Done